db/version_set.cc

```diff
@@ -403,10 +403,12 @@ bool Version::UpdateStats(const GetStats& stats) {
   FileMetaData* f = stats.seek_file;
   if (f != nullptr) {
     f->allowed_seeks--;
-    if (f->allowed_seeks <= 0 && file_to_compact_ == nullptr) {
-      file_to_compact_ = f;
-      file_to_compact_level_ = stats.seek_file_level;
-      return true;
+    if (!vset_->options_->disable_seek_autocompaction) {
```

Comment:
Just wondering if this line shouldn't be inside the if as well, in the `Apply` function we will not set the allowed seeks anymore so doesn't seem like we should be decrementing them as well.